libraries/AP_GPS/GPS_Backend.h

```diff
@@ -61,6 +61,7 @@ class AP_GPS_Backend
     void broadcast_gps_type() const;
     virtual void Write_DataFlash_Log_Startup_messages() const;
 
+    virtual bool is_healthy() const { return true; }
```

Comment:
I'm familiar with C++. My point remains that you haven't implemented *any* backend that can go individually unhealthy, so unless there are plans to do so in the immediate future, this is a pointless method. It also increases flash size by 48 bytes, and adds pointless operations. Adding methods that aren't overridden and don't do anything other then return true is pointless, and a waste of resources.